Traceback (most recent call last):
  File "snippet.py", line 32, in <module>
    import aiy.assistant.auth_helpers
ImportError: No module named aiy.assistant.auth_helpers
